.github/workflows/test-integrations-data-processing.yml

```diff
@@ -25,7 +25,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.6","3.7","3.8","3.11","3.12"]
+        python-version: ["3.6","3.7","3.8","3.10","3.11","3.12"]
```

Comment:
Why do we need to add 3.10 here?

Reply:
This is not a manual change, the CI yamls are generated by [this script](https://github.com/getsentry/sentry-python/blob/master/scripts/split-tox-gh-actions/split-tox-gh-actions.py) which parses `tox.ini` to figure out (among other things) which Python versions should be used. This is basically just the union of all Python versions used by the test suites in a specific test group.

In this case, 3.10 was added because we added Spark to the Data Processing group and according to `tox.ini`, the Spark tests [should be run against 3.10](https://github.com/getsentry/sentry-python/pull/3199/files#diff-ef2cef9f88b4fe09ca3082140e67f5ad34fb65fb6e228f119d3812261ae51449R223-R224). Apparently, no other test suite in the Data Processing group is using 3.10, which is why it wasn't in `python-version` until now.